scheduler/src/cook/kubernetes/api.clj

```diff
@@ -637,11 +637,11 @@
 
 (defn- make-empty-volume
   "Make a kubernetes volume"
-  [name]
-  (.. (V1VolumeBuilder.)
-      (withName name)
-      (withEmptyDir (V1EmptyDirVolumeSource.))
-      (build)))
+  [^String name]
+  (let [^V1VolumeBuilder builder (-> ^V1VolumeBuilder (V1VolumeBuilder.)
```

Comment:
I had several cases where {..} didn't properly propagate types, so I removed them. (Also some cases where {^Foo} within an expression also didn't propagate and avoid the reflection. Don't know why... That's the one where I setup an intermediate variable with {(let [...} so it would type.